src/building/compiler-documenting.md

```diff
@@ -1,55 +1,51 @@
 # Building documentation
 
-You might want to build documentation of the various components
-available like the standard library. There’s two ways to go about this.
-You can run rustdoc directly on the file to make sure the HTML is
-correct, which is fast. Alternatively, you can build the documentation
-as part of the build process through `x.py`. Both are viable methods
-since documentation is more about the content.
+You can run `rustdoc` directly on a file to make sure the HTML is correct,
+which runs quickly.
```

Comment:
This seems a little too concise to me. I'd like to talk that this section is to document things on rust-lang/rust like std or submodule docs at first.

Reply:
should I remove this mention... I only kept it because I found it there, and I feel it's not worth a mention

Reply:
> and I feel it's not worth a mention

Hmm, why? I think describing what this page talks about makes some sense as not all readers will read through the entire chapter.

Reply:
I meant only the part about `rustdoc` being able to be run on a single file

Reply:
I mean you removed the "You might want to build documentation of the various components available like the standard library." sentence but the introduction now sounds a little too concise.

Reply:
my latest commit is also concise, but it explains what the chapter is about... whatyouthink

Reply:
I agree with @JohnTitor, "toolchain components" seems like jargon that won't be familiar to most readers. I think "the various components available, like the standard library or compiler," would be more clear.